cg/services/run_devices/exc.py

```diff
@@ -0,0 +1,29 @@
+from cg.exc import CgError
+
+
+class PostProcessingRunFileManagerError(CgError):
+    pass
+
+
+class PostProcessingRunDataGeneratorError(CgError):
+    pass
+
+
+class PostProcessingParsingError(CgError):
+    pass
+
+
+class PostProcessingDataTransferError(CgError):
+    pass
+
+
+class PostProcessingStoreDataError(CgError):
+    pass
+
+
+class PostProcessingStoreFileError(CgError):
+    pass
+
+
+class PostProcessingError(CgError):
+    pass
```

Comment:
I think these need doc strings to explain them

Reply:
Fixed in #3575